integration/tools/validation/src/main/java/alluxio/cli/hdfs/HdfsVersionValidationTask.java

```diff
@@ -39,11 +39,12 @@
   //    Compiled with protoc 2.5.0
   //    From source with checksum d0fda26633fa762bff87ec759ebe689c
   //    This command was run using /tmp/hadoop/share/hadoop/common/hadoop-common-2.7.2.jar
-  private static Pattern sHadoopPattern =
+  private final static Pattern sHadoopPattern =
```

Comment:
@jiacheliu3 same for `sCdhPattern` and change capitalization to `HADOOP_PATTERN`